assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -218,6 +221,19 @@ function TemplateDetail() {
                 {NextButton}
               </Column>
             </ColumnContainer>
+            {relatedTemplates && (
```

Comment:
Nit, now that this is always an array, should it be `relatedTemplates.length > 0`?

Reply:
yup! thanks!